src/fix_move.cpp

```diff
@@ -277,13 +289,14 @@ FixMove::FixMove(LAMMPS *lmp, int narg, char **arg) :
   tri_flag = atom->tri_flag;
   body_flag = atom->body_flag;
   quat_atom_flag = atom->quat_flag;
+  mu_flag = (dipoleflag && atom->mu_flag);
```

Comment:
Might be good to error if the user opts to update dipoles but `atom->mu_flag` isn't set.